### PR TITLE
populates a demo dest and source after workspace login if they dont exist

### PR DESCRIPTION
**Pull request by LemonFace0309**
https://www.loom.com/share/142db05387f349898b24c6e3d498b790?sid=7db8463c-962c-4c44-b8ac-09cc8d932774

Also added an acknowledgment to login screen:
CleanShot 2024-05-27 at 12 46 32@2x
This closes [embed-473-add-disclaimer-in-demo-app-gateway-screen-to-warn-about](https://linear.app/getcensus/issue/EMBED-473/add-disclaimer-in-demo-app-gateway-screen-to-warn-about-modifying)

### CODE REVIEW

**Comment by LemonFace0309** [comment]
* **#38** <a href="https://app.graphite.dev/github/pr/sutrolabs/census-embedded-demo/38?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @LemonFace0309 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by yeldah3 — COMMENTED** [review]
I've got a UX request, lmk if it would take too much time and we can cut it if need be. But it feels weird that we silently add these connections. Like, playing with our demo app results in us adding connections to your workspace without a heads up. Honestly I think it's fine, since mainly its our sales engineers using this app, but my ask is:

Can we have a checkbox on the entry page that is default checked to true, and says "I acknowledge Census will add a sample source and destination to my workspace to better showcase this application's features"? And then only add the connections if checked?

Again, not too worried about actual customers using the app and then being angry we added connections to their workspace, but a small heads up feels nice

**Comment by LemonFace0309** [comment]
> I've got a UX request, lmk if it would take too much time and we can cut it if need be. But it feels weird that we silently add these connections. Like, playing with our demo app results in us adding connections to your workspace without a heads up. Honestly I think it's fine, since mainly its our sales engineers using this app, but my ask is:
> 
> Can we have a checkbox on the entry page that is default checked to true, and says "I acknowledge Census will add a sample source and destination to my workspace to better showcase this application's features"? And then only add the connections if checked?
> 
> Again, not too worried about actual customers using the app and then being angry we added connections to their workspace, but a small heads up feels nice

will do after this is merged in: https://linear.app/getcensus/issue/EMBED-473/add-disclaimer-in-demo-app-gateway-screen-to-warn-about-modifying

File: pages/_app.js [hunk 1]
```diff
@@ -36,7 +60,6 @@ function Application({ Component, pageProps }) {
           workspaceAccessToken={workspaceAccessToken}
           onLogOut={() => {
             setWorkspaceAccessToken(null)
-            setLoggedIn(false)
```
Comment (yeldah3) [inline pages/_app.js:39]: Why's this getting yeeted?

Comment (LemonFace0309) [reply pages/_app.js:39]: we don't use it anymore. There is no log in :p

**Pull request closed without merging**